src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/KalbachMann/Builder.hpp

```diff
@@ -0,0 +1,29 @@
+template< typename ParentBuilder >
+class Builder: 
+  public Tabulated< Data >::Builder< Builder< ParentBuilder > >,
```

Comment:
Protected as before ...

Reply:
yeah, can't get protected inheritance to work.